cloud/src/meta-service/meta_service_tablet_stats.cpp

```diff
@@ -148,8 +149,10 @@ int get_detached_tablet_stats(const std::vector<std::pair<std::string, std::stri
     }
 
     if (unexpected_size) {
-        LOG_EVERY_N(WARNING, 100) << "unexpected tablet stats_kvs, it should be 1 or 2 or 5, size="
-                                  << stats_kvs.size() << " suffix=" << ss.str();
+        DCHECK(false);
```

Comment:
DCHECK << "${some informative msg}" (maybe the same as the following log)

Reply:
ok